Marlin/example_configurations/Cartesio/Configuration_adv.h

```diff
@@ -224,6 +224,16 @@
 #define EXTRUDER_AUTO_FAN_TEMPERATURE 35
 #define EXTRUDER_AUTO_FAN_SPEED   255  // == full speed
 
+// Define a automatic fan pin to cooldown heatbed
+// 
+// When the delta temperature between current and target bed temperature is big, 
+// heatbed cooldown fan will automaticly blow to accelerate the cooldown proccess.
+// Useful when you have a PEI build surface.
+#define HEATBED_AUTO_FAN_PIN -1
```

Comment:
> I think editing HEATBED_AUTO_FAN_PIN in configuration_adv.h is more straight forward  

That is of course retained. But in addition to the options in `Configuration_adv.h`, the pins files provide the most likely pin for each accessory. In this case, the pin will have to be a 12V pin, and thus one of the MOSFET pins.

You can see how the `En_AUTO_FAN_PINS` are handled, centrally, in the main `pins.h` file:
```cpp
//
// Assign auto fan pins if needed
//
#if !defined(E0_AUTO_FAN_PIN) && defined(ORIG_E0_AUTO_FAN_PIN)
  #define E0_AUTO_FAN_PIN ORIG_E0_AUTO_FAN_PIN
#endif
#if !defined(E1_AUTO_FAN_PIN) && defined(ORIG_E1_AUTO_FAN_PIN)
  #define E1_AUTO_FAN_PIN ORIG_E1_AUTO_FAN_PIN
#endif
#if !defined(E2_AUTO_FAN_PIN) && defined(ORIG_E2_AUTO_FAN_PIN)
  #define E2_AUTO_FAN_PIN ORIG_E2_AUTO_FAN_PIN
#endif
#if !defined(E3_AUTO_FAN_PIN) && defined(ORIG_E3_AUTO_FAN_PIN)
  #define E3_AUTO_FAN_PIN ORIG_E3_AUTO_FAN_PIN
#endif
```
This style was chosen to avoid having to put blocks like this in several board pins files:
```cpp
#ifndef E0_AUTO_FAN_PIN
  #define E0_AUTO_FAN_PIN 5
#endif
#ifndef E1_AUTO_FAN_PIN
  #define E1_AUTO_FAN_PIN 5
#endif
#ifndef E2_AUTO_FAN_PIN
  #define E2_AUTO_FAN_PIN 5
#endif
#ifndef E3_AUTO_FAN_PIN
  #define E3_AUTO_FAN_PIN 5
#endif
```
…and instead they can just contain…
```cpp
#define ORIG_E0_AUTO_FAN_PIN 5
#define ORIG_E1_AUTO_FAN_PIN 5
#define ORIG_E2_AUTO_FAN_PIN 5
#define ORIG_E3_AUTO_FAN_PIN 5
```
Either style allows you to:
- turn off the pin in `Configuration_adv.h` by setting it to -1 (or empty),
- override the pin's default value by setting it to some value, or
- use the default pin set in the pins file by commenting out the define in `Configuration_adv.h`.